src/components/InformationCard1.js

```diff
@@ -2,10 +2,10 @@ import { Card, Box, Button, Avatar, Typography } from '@mui/material';
 import { useTheme } from '@mui/material/styles';
 import * as d3 from 'd3';
 import log from 'loglevel';
+import Logo from 'images/greenstand_logo.svg';
```

Comment:
@SuokN you are bringing irrelevant code into this PR, principlely, the smaller a PR is, the better. My preferred way to work with git/github is checkout the newest code ever time:
```
git fetch --all && git checkout greenstand/beta && git checkout -b issuetofix
```